devicon.json

```diff
@@ -1638,8 +1809,20 @@
           }
         ]
     },
+    {
+      "name": "embeddedc",
+      "altnames": [],
+      "tags": ["language", "programming"],
+      "versions": {
+        "svg": ["original", "original-wordmark", "plain", "plain-wordmark"],
+        "font": ["plain", "plain-wordmark"]
+      },
+      "color": "#444444",
+      "aliases": []
+    },
```

Comment:
`embeddedc` is already added. This would add it twice.

Reply:
6e9a895871f9c489551e347f4b9c05ee0830f8ae